README.md

```diff
@@ -6,16 +6,21 @@ Learning course for newbies. I'll add some stuff. I'm bad developer and do stuff
 
 ### Group #1
 
-- Maksim Fomin: fominmaksim
-- Andrew Kaenko: AndrewKaenko
-- Alex Mishenko: AlexMishenko
-- Eduard Valevich: EduardValevich
-- Ivan Lysenko: IvanLsenko
+-   Maksim Fomin: fominmaksim
+-   Andrew Kaenko: AndrewKaenko
+-   Alex Mishenko: AlexMishenko
+-   Eduard Valevich: EduardValevich
+-   Ivan Lysenko: IvanLsenko
 
 ### Group #2
-- Vladislav Pokhyl : VladFlaber
-- Roman Sokurenko: RomanSokurenko
-- Evgeniy Dotsenko: EvgenyDotsenko
-- David Korystov: DavidKorystov
-- Pavel Borisenko: PavelBorisenkoRD
-- Vasiliy Gladush: vasyagladush
+
+-   Vladislav Pokhyl : VladFlaber
+-   Roman Sokurenko: RomanSokurenko
+-   Evgeniy Dotsenko: EvgenyDotsenko
+-   David Korystov: DavidKorystov
+-   Pavel Borisenko: PavelBorisenkoRD
+-   Vasiliy Gladush: vasyagladush
+
+### Group #4
+
+-   Dmytro Tikhostup: DmytroTikhostup;
```

Comment:
Something wrong with your modification. You have changed all rows in file. It's bad experience. You have to add only one row.

Reply:
This is strange. just added one line. I will correct now